UI/css/style.css

```diff
@@ -548,26 +565,32 @@ tbody tr td ul li a{
           font-size: 14px;
      }
 
-     body {
+     * {
```

Comment:
`*` everything please re-think this

Reply:
Here I don't understand well

Reply:
Where your using, `*` it means your styling the whole DOM with a wildcat/selection. remove it and style it accordingly with the right HTML elements